configs/edison/config.B_low_res_ice_shelves_1696_JWolfe_layout_Edison

```diff
@@ -133,11 +133,8 @@ baseDirectory =  /global/project/projectdirs/acme/ACMEv0_lowres/B1850C5_ne30_v0.
 # plot on polar plot
```

Comment:
Could you add these lines in this file, right before line 46:
```
# provide an absolute path to put HTML in an alternative location (e.g. a web
 # portal)
 # htmlSubdirectory = /global/project/projectdirs/acme/www/USERNAME/RUNNAME
htmlSubdirectory = html
```

Reply:
done